tests/test_ptc.py

```diff
@@ -394,6 +405,40 @@ def test_getGoodAmps(self):
         dataset.badAmps.append("C:0,1")
         self.assertTrue(dataset.getGoodAmps() == [amp for amp in self.ampNames if amp != "C:0,1"])
 
+    def runGetGainFromFlatPair(self, correctionType='NONE'):
+        extractConfig = self.defaultConfigExtract
+        extractConfig.gainCorrectionType = correctionType
+        extractConfig.minNumberGoodPixelsForCovariance = 5000
+        extractTask = cpPipe.ptc.PhotonTransferCurveExtractTask(config=extractConfig)
+
+        expDict = {}
+        expIds = []
+        idCounter = 0
+        inputGain = self.gain  # 1.5 e/ADU
+        for expTime in self.timeVec:
+            mockExp1, mockExp2 = makeMockFlats(expTime, gain=inputGain,
+                                               readNoiseElectrons=np.sqrt(self.noiseSq),
+                                               expId1=idCounter, expId2=idCounter+1)
+            mockExpRef1 = PretendRef(mockExp1)
+            mockExpRef2 = PretendRef(mockExp2)
+            expDict[expTime] = ((mockExpRef1, idCounter), (mockExpRef2, idCounter+1))
+            expIds.append(idCounter)
+            expIds.append(idCounter+1)
+            idCounter += 2
+
+        resultsExtract = extractTask.run(inputExp=expDict, inputDims=expIds,
+                                         taskMetadata=[self.metadataContents])
+
+        for exposurePair in resultsExtract.outputCovariances:
+            for ampName in self.ampNames:
+                if exposurePair.gain[ampName] is np.nan:
+                    continue
+                self.assertAlmostEqual(exposurePair.gain[ampName], inputGain, delta=0.1)
```

Comment:
Is that large of a tolerance needed?  I would think that the simulation would get better agreement.

Reply:
I see differences of ~0.065, especially with no correction.